nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -206,7 +206,7 @@ def get_statistic_collector(
 
     @staticmethod
     def get_weight_tensor_port_ids(node: NNCFNode) -> List[Optional[int]]:
-        return [None]
+        return node.metatype.weight_port_ids
```

Comment:
As far as I understand, not all metatypes have a `weight_port_ids` attribute, then `[None]` should be returned, right?

Reply:
Yes, but weighted nodes have attribute, it's filtered before using `get_weight_tensor_port_ids`
https://github.com/openvinotoolkit/nncf/blob/fa1a4ceabee06e10f7dfb42ce12bdd9b04ed4b79/nncf/quantization/algorithms/min_max/torch_backend.py#L353-L356